src/molecules/modal/Modal.stories.js

```diff
@@ -9,6 +9,13 @@ import AInput from '../../atoms/input/Input.vue'
 import AInputWrapper from '../../molecules/input-wrapper/InputWrapper.vue'
 import AModal from './Modal.vue'
 
+const availableTransitions = [
+  'fade',
+  'slide-down',
+  'slide-right',
+  'slide'
```

Comment:
It looks like the `slide` doesn't change anything, we can remove this one from the list